lib/src/ansi_code.dart

```diff
@@ -0,0 +1,307 @@
+// Copyright 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// TODO(kevmoo) provide library documentation.
+library ansi_code;
+
+import 'dart:async';
+import 'dart:io' as io;
+
+/// Returns `true` if formatted ANSI output is enabled for [wrapWith] and
```

Comment:
[nit] `get` methods should be commented with a noun phrase rather than with "Returns"

/// Whether formatted ANSI output is enabled for...

https://www.dartlang.org/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases